app/components/list-widget/template.hbs

```diff
@@ -1,8 +1,8 @@
 <table style="width: 100%;"class="list-widget widget-body">
-    {{#each data as |item|}}
+    {{#each data as |listItem|}}
```

Comment:
I thought it was confusing for this variable to be called `item` since a parameter called `item` also gets passed into this component and is used a few lines down.